client/client-multi/client-c/src/platform-impl/Econais/EC19D/logger.c

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2015 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <sndc_sdk_api.h>
+#include "../../../platform/system_logger.h"
+
+void write_log(FILE * sink, const char * buffer, size_t message_size)
+{
+    if (!buffer) {
+        return;
+    }
+    sndc_printf(buffer);
+}
+
```

Comment:
Per code style, for readability, 3 empty lines must separate functions and other unrelated code blocks (includes, type declarations, etc.)